lib/engines/hcktest/tests.rb

```diff
@@ -349,7 +353,33 @@ def create_project_package
       @project.result_uploader.upload_file(res['hostprojectpackagepath'], r_name)
     end
 
+    def generate_summary_system_info(info)
+      "Guest system information\n  " \
+        "Host: #{info['host_name']}\n  " \
+        "OS: #{info['os_name']} #{info['os_version']}\n  " \
+        "System: #{info['system_manufacturer']} #{info['system_model']} #{info['system_type']}\n  " \
+        "CPU: #{info['processors'].join(' ')}\n  " \
+        "Memory: #{info['total_physical_memory_mb']}\n  " \
+        "BIOS: #{info['bios_version']}\n"
```

Comment:
Isn't BIOS information enough? I think the other information can be inferred from the configuration and is useful only when debugging.

Reply:
You are partially right but we still need CPU, OS and BIOS. 

When CPU is host we can't get the real value from config and QE reported some bugs related to Intel only. So at least we need know is this Intel or AMD.
OS information contains Windows version and build we need this for  testing the vnext/preview Windows version.

Reply:
The reasoning for CPU makes sense.

I'm not sure about OS. If you are *testing* vnext/preview then you should be fully aware that you are doing something unusual event without this log.

You can use heredoc here by the way.